index.html

```diff
@@ -2384,31 +2385,31 @@ <h1>WoT Building Blocks</h1>
             specific target protocols.
         </p>
         <p> The interaction implementation specifies how a <a>Thing</a>
-            actually responds to an interaction, including how state is
+            actually responds to an <a>Interaction Affordance</a>, including how state is
```

Comment:
must be "interaction" here -- it is about the actual message exchange!
BUT THESE PARAGRAPHS ARE SUPPOSED TO BE DELETED ANYWAY